### PR TITLE
Replace unittestzero with native asserts

### PR DESCRIPTION
**Pull request by bobsilverberg**
Also update versions of packages in requirements.txt

### CODE REVIEW

**Comment by bobsilverberg** [comment]
@davehunt This is the first step in updating this suite to bring it into https://github.com/mozilla/oneanddone.

r?

File: pages/page.py [hunk 1]
```diff
@@ -32,8 +31,7 @@ def __init__(self, testsetup):
     @property
     def is_the_current_page(self):
         WebDriverWait(self.selenium, 10).until(lambda s: self.selenium.title)
-        Assert.equal(self.selenium.title, self._page_title,
-                     'Expected page title: %s. Actual page title: %s' % (self._page_title, self.selenium.title))
+        assert self._page_title == self.selenium.title
         return True
```
Comment (davehunt) [inline pages/page.py]: We should just return the result of the comparison, the assert is done from the place this is called.

File: pages/page.py [hunk 2]
```diff
@@ -80,8 +78,6 @@ def wait_for_element_present(self, *locator):
         self.selenium.implicitly_wait(0)
         try:
             WebDriverWait(self.selenium, 10).until(lambda s: self._selenium_root.find_element(*locator))
-        except TimeoutException:
-            Assert.fail(TimeoutException)
```
Comment (davehunt) [inline pages/page.py]: This is weird. Let's also see if we can remove implicit waits after migrating to pytest-selenium. :smile:

File: requirements.txt [hunk 3]
```diff
@@ -1,11 +1,9 @@
 # This pulls in all the libraries needed to run Selenium tests
 # on Mozilla WebQA projects
```
Comment (davehunt) [inline requirements.txt]: Nit: I think we can get rid of the comment in this file, I've been removing it from other projects.

**Comment by davehunt** [comment]
r+ wc

File: pages/page.py (hunk 1 above)
Comment (bobsilverberg) [inline pages/page.py]: That was my first inclination too, but if we do that then the message we get with the assertion failure will just be `False is not True`, whereas if we do the equal assertion here we will see the difference between what we expected and what we got, which will be more useful for debugging.

I know I could have changed all the calls to just `is_the_current_page` without the assertion, as the assertion is being done in this method, but I feel like the intention of the call is more obvious if we are asserting that it is the current page.

What do you think?

File: pages/page.py (hunk 2 above)
Comment (bobsilverberg) [inline pages/page.py]: Sure, that's worth doing.

**Comment by bobsilverberg** [comment]
@davehunt I removed the comment from `requirements.txt`. Please respond to my response to your comment above, and then I'll know whether to merge what I have here, or make another change.

File: pages/page.py (hunk 1 above)
Comment (davehunt) [inline pages/page.py]: You make a very good point. Let's leave it as you have it here, and I'll try to keep failure traceback in mind when converting asserts.

**Comment by bobsilverberg** [comment]
Thanks @davehunt. I'm going to squash and merge.

**Comment by davehunt** [comment]
@bobsilverberg did you merge this?

**Comment by bobsilverberg** [comment]
Looks like I never did. Thanks for the reminder, @davehunt.

**Pull request merged**